src/wallabyTest.ts

```diff
@@ -17,3 +17,4 @@ TestBed.initTestEnvironment(
     BrowserDynamicTestingModule,
     platformBrowserDynamicTesting()
 );
+TestBed.initTestEnvironment = () => {};
```

Comment:
I mean what process/service is throwing the error? Wallaby doesn't - it doesn't care about type errors.